README.md

```diff
@@ -2,16 +2,14 @@
 
-The intensity of the GRB was so high, that its afterglow was detected in INTEGRAL/JEM-X almost during a week, far exceeding the previous record of a few hours. Long-lasting afterglows can also appear without prompt GRB detection in case the collimated source (jet) is not pointing straight to us (as it was also the case in GW170817). However it can be detected at a later stage when the jet decelerates and its opening angle widens. In the future, hard X-ray follow-ups of rapidly expanding population of Gravitational Waves (GW) sources will possibly reveal these so-called "orphan afterglows", providing precise localizations for sources located at distances which will be probed by the GW detectors in the coming years (z<~0.1).
+The intensity of the GRB was so high, that its afterglow was detected in INTEGRAL/JEM-X for almost a week, far exceeding the previous record of a few hours. Long-lasting afterglows can also appear without prompt GRB detection in case the collimated source (jet) is not pointing straight to us (as it was also the case in GW170817). However, it can be detected at a later stage when the jet decelerates and its opening angle widens. In the future, hard X-ray follow-ups of rapidly expanding population of Gravitational Waves (GW) sources will possibly reveal these so-called "orphan afterglows", providing precise localizations for sources located at distances which will be probed by the GW detectors in the coming years (z<~0.1).
 
-
-
-The image shows GRB 221009A lightcurve recorded by several INTEGRAL instruments. On the left - the prompt emission phase as measured by SPI-ACS, showing the brightest part of the GRB and an extended tail lasting few hours, detected even by a non-imaging detector. On the right - three revolutions of ISGRI and JEM-X observations of the hard X-ray afterglow. For the last point, only JEM-X yields a significant detection. 
+The image shows the GRB 221009A lightcurve recorded by several INTEGRAL instruments. On the lefti, before 0.1 s, there is the prompt emission phase as measured by SPI-ACS, showing the brightest part of the GRB and an extended tail lasting few hours, detected even by a non-imaging detector. On the right, afte 0.1 s, we show three revolutions of IBIS/ISGRI and JEM-X observations of the hard X-ray afterglow. For the last point, only JEM-X yields a significant detection. The inset shows the IBIS/ISGRI image of the afterglow. 
```

Comment:
it's not 0.1 seconds, obviously. It's also written on the picture

Reply:
0.1 days, yes.